tools/build_chain.sh

```diff
@@ -298,12 +315,12 @@ gen_cert_secp256k1() {
     certpath="$2"
     name="$3"
     type="$4"
-    openssl ecparam -out $certpath/${type}.param -name secp256k1
-    openssl genpkey -paramfile $certpath/${type}.param -out $certpath/${type}.key
-    openssl pkey -in $certpath/${type}.key -pubout -out $certpath/${type}.pubkey
-    openssl req -new -sha256 -subj "/CN=${name}/O=fisco-bcos/OU=${type}" -key $certpath/${type}.key -config $agpath/cert.cnf -out $certpath/${type}.csr
+    openssl ecparam -out $certpath/${type}.param -name secp256k1 2> /dev/null
+    openssl genpkey -paramfile $certpath/${type}.param -out $certpath/${type}.key 2> /dev/null
+    openssl pkey -in $certpath/${type}.key -pubout -out $certpath/${type}.pubkey 2> /dev/null
+    openssl req -new -sha256 -subj "/CN=${name}/O=fisco-bcos/OU=${type}" -key $certpath/${type}.key -config $agpath/cert.cnf -out $certpath/${type}.csr 2> /dev/null
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)